ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -90,6 +90,34 @@ import RevenueCat
         Purchases.logLevel = level
     }
 
+    @objc public static func setLogHander(completion: @escaping ([String: String]) -> Void) {
+        Purchases.logHandler = { logLevel, message in
+            let logLevelName: String
+            switch logLevel {
+            case .error:
+                logLevelName = "error"
+                break
+            case .info:
+                logLevelName = "info"
+                break
+            case .debug:
+                logLevelName = "debug"
+                break
+            case .verbose:
+                logLevelName = "verbose"
+                break
+            case .warn:
+                logLevelName = "warn"
+                break
+            }
```

Comment:
Not sure it's worth it to extract this into an extension or something like that? What do you think

Reply:
You could just use `logLevel.description`